go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/golang-jwt/jwt/v4 v4.2.0
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/stretchr/testify v1.7.0
-	github.com/ydataai/go-core v0.5.0
+	github.com/ydataai/go-core v0.5.1
```

Comment:
Let's wait for TLS support. Do you agree?

Reply:
I agree.